mitodl/micromasters
# ExamRuns: populate run for pilot period

#### What are the relevant tickets?
Part of #3085

#### What's this PR do?
This PR populates data for the pilot period into `ExamRun` and updates our exam authorization data writers to use this new structure.

#### How should this be manually tested?
- Create an `ExamAuthorization` for your user
- Run the migrations, verify your existing authorization got assigned to the pilot exam run.
- Create an exam run and a new authorization assigned for that run:
```python
from exams.factories import *
from exams.models import *
exam_run = ExamRunFactory.create(user=user)
ExamAuthorization.create(exam_run=exam_run, user=user, status=ExamAuthorization.STATUS_SUCCESS)
```
- Run this command:
```python
from exams import tasks
exams.export_exam_authorizations()
```
- Check the output and ensure it output a row for your record and the there is no `Modules` column.

#### What GIF best describes this PR or how it makes you feel?
![](https://media.giphy.com/media/13sozYO4hmSMUw/giphy.gif)
